web/src/message_view_header.js

```diff
@@ -93,12 +93,16 @@ function append_and_display_title_area(message_view_header_data) {
     }
 }
 
+function set_search_bar_text(text) {
+    $("#search_query").val(text);
+}
```

Comment:
For this sort of prep commit, it'd good to have an explanation in the commit message for why this is a good interface to extract -- e.g. some note about how the logic for setting the search bar text will get more complicated.